rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/RestPage.java

```diff
@@ -0,0 +1,47 @@
+package org.ow2.proactive_grid_cloud_portal.scheduler.dto;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+
+@XmlRootElement
+public class RestPage<T extends Serializable> {
+
+    private int size;
+
+    private ArrayList<T> list;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "list" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)